setup.py

```diff
@@ -46,30 +46,20 @@ def build_extensions(self):
         build_ext.build_extensions(self)
 
 
-extensions = [
-    Extension(
-        name="dwave.gate.simulator.simulator",
-        sources=["dwave/gate/simulator/simulator.pyx"],
-        include_dirs=["dwave/gate/simulator/"],
-        language="c++",
-    ),
-    Extension(
-        name="dwave.gate.registers.cyregister",
-        sources=["dwave/gate/registers/cyregister.pyx"],
-        include_dirs=["dwave/gate/registers/"],
-        language="c++",
-    ),
-]
-
 setup(
     name="dwave-gate",
     install_requires=[
         "numpy",
     ],
     ext_modules=cythonize(
-        extensions, annotate=bool(os.getenv("CYTHON_ANNOTATE", False))
+        ["dwave/gate/simulator/simulator.pyx",
+         "dwave/gate/registers/cyregister.pyx"
+         ],
+        annotate=bool(os.getenv("CYTHON_ANNOTATE", False))
     ),
-    include_dirs=[numpy.get_include()],
+    include_dirs=[
+        numpy.get_include(),
+        'dwave/gate/simulator/'
+        ],
     cmdclass={"build_ext": build_ext_compiler_check},
-    zip_safe=False,
```

Comment:
This package is not `zip_safe`, but I just checked, `zip_safe` is now obsolete. I guess we stopped recommending `python install ..` route?

Reply:
Oh, I removed it because it is specified in `setup.cfg`, not because it was obsolete

Reply:
Got it. Damned GiHub diffs. :laughing: